sbtx

```diff
@@ -0,0 +1,664 @@
+#!/usr/bin/env bash
```

Comment:
Do we have to add this file?
Maybe you added it by accident?

Reply:
I don't have sbt installed locally, so this wrapper helps people to start working on project ;) If you don't like wrappers I could remove it of course ;)

Reply:
Well, IMHO, a project's repository should _never_ contain stuff that is specific to any developers dev environment. It should _only_ contain things that are specific to _the project itself_.
Also, even if we wanted to add stuff like that, it has nothing to the with this PR (titled "JDK16/17 partial support").

Which bring me to another question: In which way is the solution you present here "partial"?

Reply:
We've talked about it in other comment ;)